man/systemd.network.xml

```diff
@@ -1747,6 +1747,17 @@
             Defaults to false.</para>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>UserClass=</varname></term>
+          <listitem>
+            <para>A DHCPv6 client can use UserClass option to identify the type or category of user or applications
+            it represents. The information contained in this option is a string that represents the user class of which
+            the client is a member. Each class sets an identifying string of information to be used by the DHCP
+            service to classify clients. Takes a whitespace-separated list of strings.</para>
```

Comment:
Hmm, why strings? The RFC talks about "opaque data", so it could at least in principle contain NULs. So shouldn't this be c-unescaped and allow embedded zeros?

Reply:
yes.